lib/bridge/pbn/version.rb

```diff
@@ -0,0 +1,2 @@
+require_relative '../../bridge.rb'
+module Bridge::Pbn; VERSION = "0.1.0"; end
```

Comment:
@danielevans It's the comment in bridge.rb that makes me so uneasy with the solution of module Bridge; module Pbn; VERSION="0.1.0"; end; end. What I mean to express using the namespace here is that activity is only actually taking place within a single namespace, Bridge::Pbn, and it feels like a lie to open up module Bridge right there and just shift everything I want to express over two characters.  Thanks so much for your help, your question, "what about bridge.rb?" is what did land me on this solution